schema/show.js

```diff
@@ -266,14 +266,16 @@ const ShowType = new GraphQLObjectType({
           return Image.resolve({ image_versions, image_url });
         }
 
-        return gravity(`partner/${partner.id}/show/${id}/artworks`, {
```

Comment:
Since we now have shows w/o artworks and other data, their cover image doesn't make sense (unless they have a `partner`).